### ISSUE TITLE
Loss values are not converging to a lower value.

### ISSUE THREAD

**Issue by Sanath1998** [CoinCheung/BiSeNet#331]
Hi @CoinCheung 

Thanks for this wonderful real time segmentation repo. Actually I am trying to train with the custom dataset but seems like loss values is not converging to lower value. It is converged to a value of 2.8 or so. Is this the case with you as well for custom trained dataset.

If yes, what additional changes we need to do with respect to parameters/hypermeters? Please let me know which would be very beneficial.

**Comment by CoinCheung** [comment]
Yes, the loss values are associated with number of categories and also other conditions. A value of about 2 is normal. You can draw the inference result and see the quality of your segmentation.

**Comment by CoinCheung** [comment]
Sorry for being so late, maybe this is not a problem for you now. I am closing this. Feel free to leave messages if you like.
